powerplantmatching/package_data/config.yaml

```
@@ -89,8 +89,8 @@ ENTSOE-EIC:
   fn: entsoe_eic_codes.csv
 JRC:
   reliability_score: 4
-  fn: jrc-hydro-power-database-10.zip
-  url: https://github.com/energy-modelling-toolkit/hydro-power-database/archive/refs/tags/v10.zip
+  fn: jrc-hydro-power-plant-database.csv
+  url: https://raw.githubusercontent.com/energy-modelling-toolkit/hydro-power-database/master/data/jrc-hydro-power-plant-database.csv
```

```suggestion
  url: https://raw.githubusercontent.com/energy-modelling-toolkit/hydro-power-database/7c25aef/data/jrc-hydro-power-plant-database.csv
```
Let's tie this to a particular commit. The master may change in format any time, and in absence of a tagged release, this is the next best thing.